app/views/reviews/_review.html.erb

```diff
@@ -12,11 +12,14 @@
 
   <br>
 
-    <%if current_user == review.user %>
+    <%if (current_user == review.user) %>
```

Comment:
Prefer using pundit for this, like so:
```
<% if policy(@review).update? %>
```
See documentation: https://github.com/varvet/pundit#policies

Reply:
est ce que je peux laisser le if user signed in pour l'affichage du Laissez un commentaire ? avec le policy(@review).create? je ne peux plus écrire de commentaires malgré le create en true dans mon policy review